packages/cli/test/unit/openapi/controller-spec.unit.js

```diff
@@ -15,75 +16,10 @@ describe('openapi to controllers/models', () => {
-          {
-            description: 'Returns all customers (/* customers */)',
-            comments: [
-              'Returns all customers (/* customers */)',
-              '',
-              '@param _if if condition',
-              '@param limit maximum number of results to return',
-              '@param accessToken Access token (/* access_token */)',
-              '@returns customer response',
-            ],
-            decoration: "@operation('get', '/customers')",
-            signature:
-              "async getCustomers(@param({name: 'if', in: 'query'}) _if: " +
-              "string[], @param({name: 'limit', in: 'query'}) limit: number, " +
-              "@param({name: 'access-token', in: 'query'}) accessToken: " +
-              'string): Promise<Customer[]>',
-            signatureForInterface:
-              'getCustomers(_if: string[], limit: number, accessToken: string): Promise<Customer[]>',
-            signatureForNamedParams:
-              "getCustomers(params: { if: string[]; limit: number; 'access-token': string }): Promise<Customer[]>",
-          },
-          {
-            description: 'Creates a new customer',
-            comments: [
-              'Creates a new customer',
-              '',
-              '@param _requestBody Customer to add',
-              '@param accessToken Access token (/* access_token */)',
-              '@returns customer response',
-            ],
-            decoration: "@operation('post', '/customers')",
-            signatureForInterface:
-              'createCustomer(_requestBody: Customer, accessToken: string): Promise<Customer>',
-            signatureForNamedParams:
-              "createCustomer(params: { requestBody: Customer; 'access-token': string }): Promise<Customer>",
-            signature:
-              'async createCustomer(@requestBody() _requestBody: Customer, ' +
-              "@param({name: 'access-token', in: 'query'}) accessToken: " +
-              'string): Promise<Customer>',
-          },
-          {
-            description: 'Returns a customer based on a single ID',
-            comments: [
-              'Returns a customer based on a single ID',
-              '',
-              '@param customerId ID of customer to fetch',
-              '@returns customer response',
-            ],
-            decoration: "@operation('get', '/customers/{customer_id}')",
-            signature:
-              "async findCustomerById(@param({name: 'customer_id', " +
-              "in: 'path'}) customerId: number): Promise<Customer>",
-            implementation:
-              "return {id: customerId, 'first-name': 'John', 'last-name': 'Smith'};",
-            signatureForInterface:
-              'findCustomerById(customerId: number): Promise<Customer>',
-            signatureForNamedParams:
-              'findCustomerById(params: { customer_id: number }): Promise<Customer>',
-          },
-        ],
-      },
-    ]);
+    expectToMatchSnapshot(
```

Comment:
hmm why have to stringify the snapshot here?
I noticed the other ones are like 
```ts
// provide the snapshot directly
expectFileToMatchSnapshot(controller);
```

Reply:
There are two flavors:

- Match a file to the snapshot
- Match some text to the snapshot

In controll-spec unit test, the spec is NOT written to the file system.